corehq/motech/openmrs/repeater_helpers.py

```diff
@@ -94,7 +95,7 @@ def serialize(data):
     """
     # We can get away with not worrying about namespaces because these
     # property names are fixed and unique.
-    serializers = dict(chain(ADDRESS_PROPERTIES.items(), NAME_PROPERTIES.items(), PERSON_PROPERTIES.items()))
+    serializers = dict(chain(six.iteritems(ADDRESS_PROPERTIES), six.iteritems(NAME_PROPERTIES), six.iteritems(PERSON_PROPERTIES)))
```

Comment:
Thanks for this, @nickpell . I'd like to understand the principal behind this change. Obviously the original code already works in both py2 & py3, and using `items()` in py2 is fine in this case because these are small dictionaries. ... So did you go with `six.iteritems()` because we have a policy always to use it? Or is there a different reason?

Reply:
[Modernize](https://python-modernize.readthedocs.io/en/latest/) can't tell whether or not ```chain``` assumes that its arguments are lists, so it requires you to either use ```six.iteritems``` to specify that all you need is an iterable, or to wrap items in a ```list()``` to guarantee a list is always returned.  I like using the ```six``` functions because it makes the behavior explicit between python versions, which can sometimes get more complicated than this example.